kodilla-stream/src/test/java/com/kodilla/stream/homework/ForumStatsTest.java

```diff
@@ -0,0 +1,173 @@
+package com.kodilla.stream.homework;
+
+import com.kodilla.stream.User;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static java.lang.Float.NaN;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class ForumStatsTest {
+
+    @Test
+     void testAvgAboveFortyPositiveAvg() {
+        List<User> testUsers = new ArrayList<>();
+        testUsers.add(new User("Walter White", 50, 7, "Chemists"));
+        testUsers.add(new User("Jessie Pinkman", 25, 4648, "Sales"));
+        testUsers.add(new User("Tuco Salamanca", 34, 116, "Manager"));
+        testUsers.add(new User("Gus Firing", 49, 0, "Board"));
+        testUsers.add(new User("Gale Boetticher", 44, 2, "Chemists"));
+        testUsers.add(new User("Mike Ehrmantraut", 57, 0, "Security"));
+
+        double avgAboveForty = testUsers
```

Comment:
Co tu sie dzieje? W klasie ForumStats powinny byc dwie metody, a ich nie ma. I te metody tu testowac masz :)

Reply:
ForumStats {
   private List<User> users;
   // wstrzykiwane przez konstruktor


 public double getAverageFor1() {
      return this.users.stream().
   ....
  }
   
}